src/RoME.jl

```diff
@@ -228,15 +228,17 @@ include("SpecialDefinitions.jl")
 
 ## More variable types
 # 2D
-include("variables/Point2D.jl")
-include("variables/Pose2D.jl")
-include("variables/DynPoint2D.jl")
-include("variables/DynPose2D.jl")
+# include("variables/Point2D.jl")
+# include("variables/Pose2D.jl")
+# include("variables/DynPoint2D.jl")
+# include("variables/DynPose2D.jl")
 
 # 3D
-include("variables/Point3D.jl")
-include("variables/Pose3D.jl")
+# include("variables/Point3D.jl")
+# include("variables/Pose3D.jl")
 
+#uses DFG v0.10.2 @defVariable for above
+include("variables/VariableTypes.jl")
```

Comment:
@dehann is it ok to have all variable definitions in one file like this?

Reply:
yeah, saw you did that -- think that will work well thanks otherwise they are all over the place -- good call!